arviz/plots/compareplot.py

```diff
@@ -51,9 +51,10 @@ def plot_compare(
         If None, size is (6, num of models) inches

```

Comment:
Suggested change:
```
 plot_ic_diff : bool, optional
        Plot standard error of the difference in information criteria between each model
         and the top-ranked model. Defaults to True
```

`and the top ...` is indented. Can you please remove the extra space?